packages/ui/src/components/va-aspect-ratio/VaAspectRatio.vue

```diff
@@ -21,19 +21,22 @@ const props = defineProps({
     type: [Number, String] as PropType<number | StringWithAutocomplete<'auto'>>,
     default: 'auto',
   },
-  contentHeight: { type: Number, default: 1 },
-  contentWidth: { type: Number, default: 1 },
+  contentHeight: { type: [Number, String], default: 1 },
```

Comment:
Thinking about sizes like 100px.

We should return '100px' for '100px', but 100 for 100. So if Number(prop) is NaN we should return origian lvalue.

Reply:
I believe we need to establish a boundary within the method. Otherwise, we'll have to handle percentage values, such as 80%, which are invaluable.